app/partials/recover-funds.jade

```diff
@@ -75,6 +75,14 @@
         span.help-block 
           div(ng-show="recoveryForm.confirmation.$touched")
             p(ng-show="recoveryForm.confirmation.$error.isValid" translate="NO_MATCH")
+    .flex-center.flex-end.mtm.mbl
+      input#agreement_accept.pull-right(
+        name="agreement"
+        type="checkbox"
+        ng-model="fields.acceptedAgreement"
```

Comment:
This is where it's copied from: https://github.com/blockchain/My-Wallet-V3-Frontend/blob/460cd883e031face9a9a865df8945a25c37a0f80/app/partials/signup.jade#L63-L67